City-of-Helsinki/drupal-helfi
# UHF-1172: Siteimprove module & config

How to test:

1. Checkout this branch and run `make drush-cim drush-cr`
2. Log in as admin and view any content page
3. See the Siteimprove dialog on the right hand side of the page (there are errors, more of them below)

NOTE

This is currently not working as it is installed in local/dev environment, and the Siteimprove is configured to work in `https://www.hel.fi`. We need to verify the functionality once in production.
